llvm/test/CodeGen/MIR/AMDGPU/machine-function-info-no-ir.mir

```diff
@@ -578,3 +578,18 @@ body:             |
     SI_RETURN
 
 ...
+---
```

Comment:
Should add an error test where the flag name is unrecognized

Reply:
I've put it in the generic test llvm/test/CodeGen/MIR/Generic/register-flag-error.mir